In-house patch to remove the reference to glance_store's tests in the
Python package's entry_points.txt file. Without this change, a warning
is printed in some of the Glance service logs. Not suitable for the
upstream.

--- glance_store-0.1.10/setup.cfg.orig	2014-11-17 12:15:39.000000000 -0800
+++ glance_store-0.1.10/setup.cfg	2015-01-27 09:38:35.386243430 -0800
@@ -33,8 +33,6 @@ glance_store.drivers =
 	cinder = glance_store._drivers.cinder:Store
 	gridfs = glance_store._drivers.gridfs:Store
 	vmware = glance_store._drivers.vmware_datastore:Store
-	# TESTS ONLY
-	no_conf = glance_store.tests.fakes:UnconfigurableStore
 	# Backwards compatibility
 	glance.store.filesystem.Store = glance_store._drivers.filesystem:Store
 	glance.store.http.Store = glance_store._drivers.http:Store
